src/interfaces/taskDetails.type.ts

```diff
@@ -0,0 +1,17 @@
+export type taskDetailsDataType = {
```

Comment:
I think this should  be an interface

Reply:
As I am **NOT EXTENDING** this by declaring it a second time anywhere, I have used the `type` here.